JenkinsTerraform/jenkins-agent02-installs.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+#################### Install Java: ####################
+sudo apt-get update -y
+sudo apt-get install -y default-jre
+sudo apt-get update -y
+
+#################### Install kubectl: ####################
+
+#################### Update the apt package index and install packages needed to use the Kubernetes apt repository ####################
+sudo apt-get update
+# apt-transport-https may be a dummy package; if so, you can skip that package
+sudo apt-get install -y apt-transport-https ca-certificates curl
+
+#################### Download the public signing key for the Kubernetes package repositories ####################
+
+curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+
+#################### Add the appropriate Kubernetes apt repository ####################
+#################### This overwrites any existing configuration in /etc/apt/sources.list.d/kubernetes.list ####################
+echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
+
+#################### Update apt package index, then install kubectl: ####################
+sudo apt-get update
+sudo snap install kubectl
+sudo apt-get install -y kubectl
+
+#################### Ensure the version you installed is up-to-date: ####################
+kubectl version --client
+
+#################### Installation complete ####################
+echo "-----------------------kubectl successfully installed!-----------------------"
+
+
+
```

Comment:
```suggestion
curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp

sudo mv /tmp/eksctl /usr/local/bin

eksctl version
```
You also need to install `eksctl`